cmd/jag/commands/scan.go

```diff
@@ -18,24 +21,32 @@ import (
 )
 
 const (
-	scanTimeout = 600 * time.Millisecond
-	scanPort    = 1990
+	scanTimeout  = 600 * time.Millisecond
+	scanPort     = 1990
+	scanHttpPort = 9000
 )
 
 func ScanCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "scan",
+		Use:   "scan [address]",
```

Comment:
Maybe we can use this if the device option (-d) is an address?

Reply:
Will do a follow-up PR.

Reply:
https://github.com/toitlang/jaguar/pull/196